src/js/components/TextInput/TextInput.js

```diff
@@ -129,6 +132,25 @@ const TextInput = forwardRef(
     );
 
     const [suggestionsAtClose, setSuggestionsAtClose] = useState();
+    const [tipContent, setTipContent] = useState(
```

Comment:
```suggestion
    const [tip, setTip] = useState(
```
would this be sufficient?

Reply:
done